File: Assets/Smash Way/Scripts JS/GameOther/DestroyAll.js
#pragma strict

var Cam_Object:GameObject;
var Distance:float;
//var AudAlvl:AudioClip;
private var Rest:int=0;

function Search(){
yield WaitForSeconds(2);
if(gameObject.transform.position.z<Cam_Object.transform.position.z-Distance){
Destroy(gameObject);
}else{Rest=0;}
}

function Update(){
if(Rest==0){
Rest=1;
Search();
}
}

File: Assets/Smash Way/Scripts CS/GameOther/DestroyAll.cs
using UnityEngine;
using System.Collections;

public class DestroyAll : MonoBehaviour {



	public GameObject Cam_Object;
	public float Distance;
	private int Rest = 0;

	IEnumerator Search(){
		yield return new WaitForSeconds(2);
		if(gameObject.transform.position.z < Cam_Object.transform.position.z- Distance){
			Destroy(gameObject);
		}else{Rest=0;}
	}

	void Update(){
		if(Rest == 0){
			Rest= 1;
			StartCoroutine("Search");
		}
	}

}
